crates/cluster/src/node_operator.rs

```diff
@@ -62,7 +62,7 @@ impl Name {
 pub struct Data(pub(crate) Vec<u8>);
 
 /// Entity operating a set of [`Node`]s within a WCN cluster.
-#[derive(AsRef, Clone, Debug)]
+#[derive(AsRef, Clone, Debug, Serialize)]
```

Comment:
May as well remove `Serialize` as it won't be used.

Reply:
it's used by tests but can create a test-only type to address that